### PR TITLE
[CP-2040] Character sequence causes file being displayed as empty lin…

### PR DESCRIPTION
**Pull request by lukasmas**
…e in relaxation app

Fixed problem with displaying some filenames in Relaxation

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

**Review by dawidwojtas-mudita — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationMainWindow.cpp [hunk 1]
```diff
@@ -28,10 +28,18 @@ namespace gui
 
     void RelaxationMainWindow::setSoundsList(std::vector<db::multimedia_files::MultimediaFilesRecord> sounds)
     {
+        auto trimTittle = [](const std::string &title) {
+            std::string shortTittle = title.substr(0, maxDisplayedTitleLength);
+            if (shortTittle.back() == '\342') {
```
Comment (exeosdev) [inline products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationMainWindow.cpp:33]: I'd define properly named constant here and use that for comparison. It's now a bit unclear what does this magic number mean and why is it removed specifically.

File: products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationRunningLoopWindow.cpp [hunk 2]
```diff
@@ -77,6 +77,9 @@ namespace
             return title;
         }
         std::string newTittle = title.substr(0, maxPossibleCharsToDisplay - 2);
+        if (newTittle.back() == '\342') {
```
Comment (exeosdev) [inline products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationRunningLoopWindow.cpp:80]: Same here

**Review by exeosdev — CHANGES_REQUESTED** [review]

File: products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationMainWindow.cpp (hunk 1 above)
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationMainWindow.cpp]: Done.

File: products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationRunningLoopWindow.cpp (hunk 2 above)
Comment (lukasmas) [reply products/BellHybrid/apps/application-bell-relaxation/windows/RelaxationRunningLoopWindow.cpp]: Done.

**Review by exeosdev — APPROVED** [review]

**Pull request merged**